CHANGES.rst

```diff
@@ -25,6 +25,8 @@ Specviz2d
 API Changes
 -----------
 
+- Extend unix style wildcard matching in ``user_api`` for ``SelectPluginComponent`` objects. [#3744]
```

Comment:
I think this might use a bit too much internal-language (feel free to revise to something else though).

```suggestion
- Extend unix style wildcard matching for select objects supporting multiselect. [#3744]
```